pybliometrics/scopus/serial_title.py

```diff
@@ -163,14 +169,18 @@ def __init__(self, issn, refresh=False, view="ENHANCED"):
                              ', '.join(allowed_views))
         # Load json
         self._id = str(issn)
-        Retrieval.__init__(self, identifier=self._id, view=view,
+        self._years = years
+        # force refresh when years is specified
+        if years is not None:
+            refresh = True
+        Retrieval.__init__(self, identifier=self._id, view=view, date=years,
                            api='SerialTitle', refresh=refresh)
         self._json = self._json['serial-metadata-response']
         self._entry = self._json['entry'][0]
 
     def __str__(self):
         """Print a summary string."""
-        date = self.get_cache_file_mdate().split()[0]
+        dateCACHE = self.get_cache_file_mdate().split()[0]
```

Comment:
Since the new parameter has been renamed to `years`, this one can go back to normal